Model/src/BoardPackage/RailRoadSquare.java

```diff
@@ -1,7 +1,162 @@
 package BoardPackage;
 
-/**
- * Created by unclear on 2/5/18.
- */
-public class RailRoadSquare {
+import GamePackage.Player;
+
+/*****************************************************************
+ The RailRoadSquare class is a BoardSquare that tracks
+ information specific to RailRoadSquares.
+
+ @author Dylan Kernohan
+ @version 2/12/2018
+ *****************************************************************/
+public class RailRoadSquare extends BoardSquare{
+
+    /**The price of the RailRoad square*/
+    private final int PRICE;
+
+    /**The initial rent of the RailRoad square*/
+    private final int BASE_RENT;
+
+    /**The rent if a player owns two RailRoads*/
+    private final int TWO_RENT;
+
+    /**The rent if a player owns three RailRoads*/
+    private final int THREE_RENT;
+
+    /**The rent if a player owns four RailRoads*/
+    private final int FOUR_RENT;
+
+    /**The value a player receives if they mortgage this RailRoad*/
+    private final int MORTGAGE_VALUE;
+
+    /**A flag to tell if this RailRoad is currently mortgaged*/
+    private boolean isMortgaged;
+
+    /**The player that owns this RailRoad*/
+    private Player owner;
+
+
+    /*****************************************************************
+     Constructor instantiates the variables and sets the name of the
+     BoardSquare to what is input as a parameter.
+
+     @param name String that will be the name of the BoardSquare.
+     @param price The price of the RailRoad square
+     @param baseRent The initial rent of the RailRoad square
+     @param twoRent rent if a player owns two RailRoads
+     @param threeRent rent if a player owns three RailRoads
+     @param fourRent The rent if a player owns four RailRoads
+     @param mortgageValue The value a player receives if they
+     mortgage this RailRoad
+     ****************************************************************/
+    public RailRoadSquare(String name, int price, int baseRent,
+                          int twoRent, int threeRent, int fourRent,
+                          int mortgageValue) {
+
+        super(name);
+        this.PRICE = price;
+        this.BASE_RENT = baseRent;
+        this.TWO_RENT = twoRent;
+        this.THREE_RENT = threeRent;
+        this.FOUR_RENT = fourRent;
+        this.MORTGAGE_VALUE = mortgageValue;
+        this.isMortgaged = false;
+        // TODO: Do we want to keep this null or make Bank own all squares to start out with? If so we would add Bank as param
```

Comment:
remove this comment. Next time you can just do the comment through the PR instead of the actual code.